data/json/obsolete.json

```diff
@@ -497,69 +497,6 @@
     },
     "flags": [ "BOMB", "TRADER_AVOID" ]
   },
-  {
-    "id": "tool_anfo_charge",
```

Comment:
This and following item were obsoleted in https://github.com/CleverRaven/Cataclysm-DDA/pull/39492, so you probably need to delete these objects from `\data\json\items\tool\explosives.json`.

Reply:
ok, I'll bring back these and delete them in `explosives.json` a bit later